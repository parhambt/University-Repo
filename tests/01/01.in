create_template 'Simulator' Math:hard:1 Chemistry:easy:1 Physics:medium:2
create_template 'Simulator' Geography:medium:2
generate_test 'test No.1' 'Simulater'
generate_test 'test No.1' 'Simulator'
generate_test 'test No.2' 'Simulator'
attend 'test No.3'
attend 'test No.1'
previous
2
3
previous
2

2
auto_generate 'test No.3'
attend 'test No.3'
1
2
2
previous
previous
3
2
4
3
2
3
3
4
1
report tests
attend 'test No.2'
1
2
3
4
report subject Physics
