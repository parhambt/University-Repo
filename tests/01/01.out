misagh has done 0 tasks.
misagh is currently not working on any tasks.
